libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -176,6 +176,21 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS", 21, AP_BattMonitor_Params, _options, 0),
 
+    // @Param: CELL_DET_V
+    // @DisplayName: Maximum cell voltage for cell count detection
+    // @Description:
+    // @Units: V
+    // @Increment: 0.01
+    // @User: Advanced
+    AP_GROUPINFO("CELL_DET_V", 22, AP_BattMonitor_Params, _cell_detect_volt, 4.25f),
```

Comment:
Name too long

Reply:
Doesn't look like that's the problem. I remember seeing a message saying when they are too long. Also I shorten them and still the same problem